src/main/java/redis/clients/jedis/exceptions/JedisValidationException.java

```diff
@@ -0,0 +1,21 @@
+package redis.clients.jedis.exceptions;
+
+/**
+ * A validation error.
+ */
+public class JedisValidationException extends JedisException {
```

Comment:
I thought it was odd that one didn't already exist. Validation exceptions are normal. I don't think making it MultiCluster specific makes much sense since its not in anyway related to multi-clusters, it just so happens that I am the first to validate instead of allowing things to just blow up due to incorrect configuration. I think you should reconsider this ask but if you feel strongly I can rename it.